README.md

```diff
@@ -0,0 +1,54 @@
+# cron-admin
+Web interface and API for managing a cron table stored in a database. For details on cron times and conventions see [this reference](http://www.adminschoice.com/crontab-quick-reference).
+
+## API Endpoints
+
+#### GET: `/active-jobs`
+
+Returns a list of distinct job function names which are actively scheduled.
+
+#### GET: `/jobs`
+
+Query param:
+* Function
+
+Returns a list of all jobs associated with the given function.
+
+#### POST: `/jobs`
+
+Body params:
+* Function
+* CronTime
+* Workload
+* TimeZone (optional, defaults to "America/Los\_Angeles")
+
+Posts a new, active job. Returns error if "Function" is null or "CronTime" is an invalid format.
+
+#### DELETE: `/jobs/{job_id}`
+
+Removes job from the database.
+
+#### PUT: `/jobs/{job_id}`
+
+Body params (all optional):
+* IsActive
+* CronTime
+* Workload
```

Comment:
I'd make these all required--basically it should be the some payload that you send to a POST

see "an API should define a full object “PUT” in Ben's inter-service API spec